CHANGELOG.md

```diff
@@ -7,11 +7,19 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Enhancements
 
-* Swift: A `write` block may now `throw`, reverting any changes already made in the transaction.
+* Swift: A `write` block may now `throw`, reverting any changes already made in
+  the transaction.
+* Reduce address space used when committing write transactions.
+* Significantly reduce the download size of prebuilt binaries and slightly
+  reduce the final size contribution of Realm to applications.
```

Comment:
The changes to util::terminate slightly reduced the linked size (but not by a very exciting amount).